### DISCUSSION TITLE
Currently 'queued' command before execution

### DISCUSSION THREAD

**Discussion by Yair-Ziv1M** [nvim-lualine/lualine.nvim#1329] [Q&A]
Not really the best explanation in the title
I removed the default vim cmd line but the only feature I am missing is the queued command being shown, like seeing 'da' in the bottom before pressing '(' or smth to delete around brackets, is there a way to see that in the lualine?

**Comment by nyngwang** [comment]
From the [comment](https://www.reddit.com/r/neovim/comments/1jlemyn/comment/mk8pxzi/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) on reddit:

> Figured it out.
> ```lua
> sections = { lualine_a = {"%S"} }
> ```
> Mentioned in: `:h lualine-Custom-components`. See also: `:h 'statusline'`
> Also, i believe you need to do
> ```vimscript
> :set showcmdloc=statusline
> ```

**Reply by nyngwang** [reply]
@Yair-Ziv1M I think we can safely close this thread~ :D

**Reply by Yair-Ziv1M** [reply]
Agreed
